.github/workflows/main.yml

```diff
@@ -5,17 +5,13 @@ on: [push]
 jobs:
   run-scrapers-tests:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node: ["12", "13"]
-      fail-fast: false
     name: Scraper Tests - Node ${{ matrix.node }} compatibility
     steps:
       - uses: actions/checkout@v1
       - name: Setup node
         uses: actions/setup-node@v1
         with:
-          node-version: ${{ matrix.node }}
+          node-version: "16"
```

Comment:
v13 compatibility was failing cause the `concurrently` package that this PR adds isn't compatible with v13. Thought It's probably best to go up to v16 anyways especially since our code works on v16?

Reply:
tbh I think node v16 has a lot of issues... not sure if it's very feasible to change that to v16 maybe @bugsalexander might have more thoughts on this

Reply:
oh I didn't know that, thought it was fine since our tests pass and the current LTS version of node is v16

Reply:
hmm I guess but I am also just thinking whether or not v16 might cause problems for some incompatibility with some packages we have in the codebase

Reply:
Yeap that makes sense but this is also just the node version for CI running our scraper tests(which work). Also if packages are incompatible with v16 then the `yarn install` would complain. And from some quick googling node actually hasn't had breaking changes for the past 2 major releases. Oh and we're kinda redoing the scrapper anyways right lmao?

Reply:
lol that's true

Reply:
i'd probably stick with 14 for now since it's battle tested, we can always bump version to 16 later?

Reply:
we should probably use the same node version for everything (frontend, scrapers, backend) so i'd say use 14 for now since we don't know if all will work on 16.